### PR TITLE
[RFR] Implement Dashboard view

### PR DESCRIPTION
**Pull request by jeromemacias**

### CODE REVIEW

File: app/Stores/DashboardStore.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                        response.result.data
+                    );
+
+                    dataStore.setEntries(
+                        panel.get('entity').uniqueId,
+                        entries
+                    );
+                });
+
+                this.data = this.data.update('panels', v => panels);
```
Comment (manuquentin) [inline app/Stores/DashboardStore.js]: Can we update multiple entries in one time ?

Comment (jeromemacias) [inline app/Stores/DashboardStore.js]: I don't think so.

**Comment by manuquentin** [comment]
Needs rebase

**Comment by manuquentin** [comment]
Can you add some test ?

**Comment by jeromemacias** [comment]
I propose to add tests in another PRs, so we can use this code as a base for next tasks.

**Comment by manuquentin** [comment]
Deal

**Pull request merged**